airbyte-integrations/connector-templates/source_acceptance_test_files/acceptance-test-docker.sh

```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env sh
 
+# Set this variable to false to bypass spec backward compatibility checks
+RUN_SPEC_BACKWARD_COMPATIBILITY_TEST=true
+
 # Build latest connector image
 docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2-)
 
-# Pull latest acctest image
+# Pull latest source-acceptance-test image
 docker pull airbyte/source-acceptance-test:latest
 
+TEST_OPTIONS="--acceptance-test-config /test_input"
+if [ "$RUN_SPEC_BACKWARD_COMPATIBILITY_TEST" = false ] ; then
+    echo "Won't run spec backward compatibility tests"
+    # Add a -m flag to pytest options to not run tested with spec_backward_compatibility marker
+    TEST_OPTIONS+='-m "not spec_backward_compatibility"'
+fi
```

Comment:
This will only work if the SAT is being run locally through this script. On CI we run the gradle task defined here https://github.com/airbytehq/airbyte/blob/b73713e610cc0a588ec6e14d01f03efbe331f345/buildSrc/src/main/groovy/airbyte-source-acceptance-test.gradle

This means that we won't be able to test/publish connectors that are intentionally breaking compatibility tests. I would suggest instead adding a configuration option as part of the acceptance test config to skip these, as is done in a few other spots, like `expect_trace_message_on_failure` and `skip_comprehensive_incremental_tests`.

Also, it would be great if rather than just a boolean (skip/dont skip) we could specify which "previous version" we want to skip against so we don't forget to re-enable.

Reply:
Cool idea thanks for the suggestion!

Reply:
@pedroslopez I implemented your suggestion: 
* Create a new pydantic model: `BackwardCompatibilityTestsConfig` with two fields: `previous_connector_version` (defaults to `latest`) and `run_backward_compatibility_tests` (defaults to `True`)
* Use this model as a field for `SpecTestConfig`